src/components/CardPlay/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useRef, useEffect } from "react";
+import { PlayCircleTwoTone, PauseCircleTwoTone } from '@ant-design/icons';
+import "./styles.css";
+
+    const CardPlay = (props) => {
```

Comment:
Presta atención a la indentación.
```suggestion
const CardPlay = (props) => {
```